adapt-translations-from-smartling.js

```diff
@@ -0,0 +1,61 @@
+#!/opt/nodenv/shims/node
+
+/**
+ * This file is used by DDG employees to integrate translations coming from Smartling.com into this repo.
+ * Community translations do not need to use this files.
+ */
+
+const fs = require('fs');
+const path = require('path');
+const readline = require('readline');
+const rl = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const copyFiles = (srcDir, destDir) => {
+  const files = fs.readdirSync(srcDir)
+  return files.map(file => {
+    const destFile = path.join(destDir, file.replace('.pot', '.po'));
+    const srcFile = path.join(srcDir, file);
+    try {
+      const dataBuffer = fs.readFileSync(srcFile);
+      const fileContent = dataBuffer.toString();
+      fs.writeFileSync(destFile, fileContent);
+      console.log(`Copied file from ${srcDir} to ${destFile}`);
+    } catch (e) {
+      console.log(`Got an error trying to copy the file ${srcDir}: ${e.message}`)
+    }
+  })
+}
+
+const start = () => {
+  try {
+    rl.question('Where are the translations? ', function(baseFolder) {
+      // Needed because drag-dropping a path in the terminal on macOS adds a space at the end.
+      baseFolder = baseFolder.trim();
```

Comment:
@GioSensation for me in MacOS, drag-dropping the folder added an extra `'` at the start and end of the path, which caused the script to fail (e.g. `'/Users/myuser/duckduckgo.pot'` instead of `/Users/myuser/duckduckgo.pot`.

We can cover this case with e.g. `baseFolder.trim().replace(/^'/, '').replace(/'$/, '');`.